openmc/mesh.py

```diff
@@ -1514,13 +1542,21 @@ class SphericalMesh(StructuredMesh):
 
     """
 
-    def __init__(self, mesh_id=None, name=''):
+    def __init__(
+        self,
+        mesh_id: typing.Optional[int] = None,
+        name: str = '',
+        origin: typing.Tuple[float] = (0., 0., 0.),
```

Comment:
Same for the type hinting on this origin as well, perhaps we should have the default values as an np.array([0, 0, 0]) and the type hint as an np.array